packages/mongoose-store/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
I noticed we have mit license - I know that this is demo repo but best to be aware and make sure what license we should use.